public/chore.js

```diff
@@ -30,4 +30,22 @@ choreList.forEach(function (chore) {
 });
 
 const app = document.querySelector('#app');
-app.appendChild(list);
\ No newline at end of file
+app.appendChild(list);
+
+const newChore = {
+    name: document.getElementById('chore').value,
+    assigned: document.getElementById('assigned').value
+}
+
+
+function addChore() {
+const newChore = {
+    name: document.getElementById('chore').value,
+    assigned: document.getElementById('assigned').value
+}
+    Object.create(newChore);
```

Comment:
You don't need to `Object.create()` here because you aren't basing chore off a pre-existing model. So line 46 can just be removed. We can walk through this more if you had a reason you thought it was necessary.